plugin/src/main/java/com/denizenscript/denizen/objects/InventoryTag.java

```diff
@@ -2303,6 +2304,22 @@ else if ((object.inventory instanceof FurnaceInventory)) {
             }
             return new ElementTag(BukkitScriptEvent.tryInventory(object, attribute.getParam()));
         });
+
+        // <--[tag]
+        // @attribute <InventoryTag.viewers>
+        // @returns ListTag(PlayerTag)
+        // @description
+        // Returns a list of players viewing the inventory.
+        // -->
+        tagProcessor.registerTag(ListTag.class, "viewers", (attribute, object) -> {
+            ListTag list = new ListTag();
+            for (HumanEntity viewer : object.getInventory().getViewers()) {
+                if (!EntityTag.isNPC(viewer)) {

```

Comment:
going to fix myself but just to point out ^ `viewer instanceof Player` check should be here, in part for formality reasons, in part because it's actually possible for a HumanEntity to not be a Player in certain very rare weird cases